src/components/Nav.js

```diff
@@ -1,21 +1,27 @@
-import { NavLink } from 'react-router-dom';
+import { ReactComponent as ListIcon } from '../assets/list-icon.svg';
+import { ReactComponent as MapIcon } from '../assets/map-icon.svg';
+import { Button } from '../components/Button';
 
 const Nav = () => (
   <nav>
     <ul className="nav">
       <li>
-        <NavLink exact to="/">
-          Map
-        </NavLink>
+        <Button
+          href="/"
+          text="Map"
+          className="btn btn__link"
+          icon={MapIcon}
+          label="Map button"
+        />
       </li>
       <li>
-        <NavLink to="/list">List</NavLink>
-      </li>
-      <li>
-        <NavLink to="/settings">Settings</NavLink>
-      </li>
-      <li>
-        <NavLink to="/help">Help</NavLink>
+        <Button
```

Comment:
Awesome! Great change 👍 !

Reply:
It feels weird to do the link -> button thing, but after reading a few accessibility articles written in the last year, I didn't feel as bad, as long as it is labeled well!